pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -639,6 +748,108 @@ const New = () => {
+                <h2 className='mt-8'>Add Choices</h2>
+                {multiChoiceForm.options.map((option, index) => {
+                  // copy index to keep its value for onChange function
+                  const odx = index;
+
+                  return (
+                    <div key={odx} className="mb-3 mt-3 border border-fgd-4 p-4 md:p-6 rounded-lg">
+                    <div className="flex flex-row justify-between">
+                      <h2 className='mb-4'>Choice {odx + 1}</h2>
+                      {odx > 1 ?
+                        <LinkButton
+                          className="flex font-bold items-center ml-4 text-fgd-1 text-sm"
+                          onClick={() => updateOptions(odx, option, false)}
+                        >
+                          <XCircleIcon className="h-5 mr-1.5 text-red w-5" />
+                            Remove
+                        </LinkButton>
+                      : ""}
+                    </div>
+                    <StyledLabel>Add a Label</StyledLabel>
+                    <div className='text-sm font-extralight text-fgd-3 mt-1 mb-2 max-w-lg'>
+                      This is the text voters will see when they vote.
+                    </div>
+                    <Input
+                      placeholder={`Voting Choice ${odx + 1}`}
+                      value={option}
+                      type="text"
+                      error={
+                        !option.length && isMultiFormValidated ? "The option can't be empty" : ""
+                      }
+                      showErrorState={option.length === 0 && isMultiFormValidated}
+                      onChange={(event) => updateOptions(odx, event.target.value, true)}
+                      disabled={option === "None of the Above"}
+                    />
+                  </div>)}
+                )}
+                <div className="flex flex-row justify-between">
+                  <div>
+                    <LinkButton
+                      onClick={() => updateOptions(-1, "", true)}
+                      disabled={multiChoiceForm.options.length > 9}
+                      className='flex flex-row items-center gap-2 font-bold pt-2'
+                    >
+                        <AddAlt className='text-green'/>
+                        <div>Add another voting choice</div>
+                    </LinkButton>
+                  </div>
+                  <div>
+                    <LinkButton
+                      onClick={() => handleNotaButton()}
+                      disabled={hideNotaButton || multiChoiceForm.options.length > 9}
+                      className='flex flex-row items-center gap-2 font-bold pt-2'
+                    >
+                        <AddAlt className='text-green'/>
+                        <div>Add &apos;None of the Above&apos; choice</div>
+                    </LinkButton>
+                  </div>
+                </div>
+              </div>
+              :
+              !isMulti ?
```

Comment:
`isMulti` is known to be false here so this ternary operator is pointless.
I need to find an eslint rule for this tbh.

Reply:
Earlier `isMulti` could have `undefined` as a possible value. I forgot to remove this check. Now corrected.